index.html

```diff
@@ -435,11 +438,11 @@ <h3>Instantiate Cryptosuite</h3>
 If |options|.|cryptosuite| is `eddsa-rdfc-2022`:
             <ol class="algorithm">
               <li>
-Set |cryptosuite|.|createProof| to the result of running the algorithm in Section
+Set |cryptosuite|.|createProof| to the algorithm in Section
```

Comment:
I think these changes are incorrect. The algorithm doesn't go into `|cryptosuite|.|createProof|`; its result does. 
```suggestion
Set |cryptosuite|.|createProof| to the algorithm in Section
```

Reply:
No,  this  is  setting  up the cryptosuite. So  it is setting up the algorithms (functions) to be  called. See  how  this is  done in  https://w3c.github.io/vc-di-ecdsa/#instantiate-cryptosuite.  This is typical of  "pluggable crypto" APIs.

Reply:
@Wind4Greg — OK, I see the distinction. Thanks for the pointer and explanation!